src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs

```diff
@@ -1596,113 +1596,115 @@ private async Task ReceiveLoop(ConnectionState connectionState)
         var timer = new TimerAwaitable(TickRate, TickRate);
         var timerTask = connectionState.TimerLoop(timer);
 
-        var uploadStreamSource = new CancellationTokenSource();
-        connectionState.UploadStreamToken = uploadStreamSource.Token;
-        var invocationMessageChannel = Channel.CreateUnbounded<InvocationMessage>(_receiveLoopOptions);
+        using (var uploadStreamSource = new CancellationTokenSource())
```

Comment:
Use the `using var name = new type();` syntax. You don't need the parenthesis and additional formatting.
https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0063

Reply:
Thank you for the feedback, Brennan! I've updated the code to use the using var syntax as suggested. Please let me know if there's anything else that needs adjustment.